src/Common/src/Interop/Unix/System.Net.Security.Native/SecuritySafeHandles.cs

```diff
@@ -13,16 +13,65 @@ namespace System.Net.Security
     internal sealed class SafeFreeNegoCredentials : SafeFreeCredentials
     {
         private SafeGssCredHandle _credential;
+        private readonly bool _isNtlm;
+        private readonly string _userName;
+        private readonly bool _isDefault;
 
         public SafeGssCredHandle GssCredential
         {
             get { return _credential; }
         }
 
-        public SafeFreeNegoCredentials(string username, string password, string domain) : base(IntPtr.Zero, true)
+        public bool IsNtlm
```

Comment:
In the parameters early you used "isNtlmOnly" and "isNtlmUsed".  Here you just have "IsNtlm".  Sorry if I already asked this, but what's the meaning of the three different terms?